### PR TITLE
Add release info logs to host-agent container

### PR DESCRIPTION
**Pull request by apoorva11029**
git commit ID and build time info will be present in the container logs for host-agent and controller

### CODE REVIEW

File: cmd/controller/main.go [hunk 1]
```diff
@@ -96,6 +96,13 @@ func main() {
 		panic(err.Error())
 	}
 
+	if controller.GetVersion().GitCommit != "" {
```
Comment (gaurav-dalvi) [inline cmd/controller/main.go]: We are checking exact same thing here as well : https://github.com/noironetworks/aci-containers/blob/36fd98d6054b94c154c7facf1e5598707c167ec0/cmd/controller/main.go#L44 
Why do we need to repeat this ?

File: cmd/hostagent/main.go [hunk 2]
```diff
@@ -80,6 +80,13 @@ func main() {
 		panic(err.Error())
 	}
 
+        if hostagent.GetVersion().GitCommit != "" {
+                versionInfo := hostagent.GetVersion()
+                log.Info("Running controller built from git commit ID " +
```
Comment (snaiksat) [inline cmd/hostagent/main.go]: controller -> host-agent ?

File: pkg/controller/controller.go [hunk 3]
```diff
@@ -275,10 +275,6 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 	var privKey []byte
 	var apicCert []byte
 
-	if gitCommit != "" {
```
Comment (gaurav-dalvi) [inline pkg/controller/controller.go]: Is there any way in the logs developer can see the version / commit id after removing this ?

File: cmd/hostagent/main.go (hunk 2 above)
Comment (apoorva11029) [reply cmd/hostagent/main.go]: Right, fixed.

File: cmd/controller/main.go (hunk 1 above)
Comment (apoorva11029) [reply cmd/controller/main.go]: Need it for the version flag too

File: pkg/controller/controller.go (hunk 3 above)
Comment (apoorva11029) [reply pkg/controller/controller.go]: The (equivalent)code has been moved to cmd/controller/main.go

**Pull request merged**